src/services/redisSessionStorage.ts

```diff
@@ -308,6 +284,45 @@ export default class RedisSessionStorage extends RedisStorageUtils
     );
   }
 
+  public async userHasActiveSessions(
+    fiscalCode: FiscalCode
+  ): Promise<Either<Error, boolean>> {
+    const sessionKeys = await this.readSessionInfoKeys(fiscalCode);
+    if (sessionKeys.value === sessionNotFoundError) {
+      return right(false);
+    } else if (isLeft(sessionKeys)) {
+      return left(sessionKeys.value);
```

Comment:
```suggestion
      return sessionKeys;
```

Reply:
Herewith this code chage will be a type error:
```
Type 'Left<Error, readonly string[]>' is not assignable to type 'Left<Error, boolean>'
```
A cast is necessary like this `return sessionKeys  as Left<Error, boolean>`
What do you suggest @gunzip ?

Reply:
keep your version